src/murzynek-cocoa-cake.md

```diff
@@ -0,0 +1,35 @@
+# Cocoa Cake - Murzynek
+
+![Cocoa Cake - Murzynek](pix/murzynek.webp)
+
+A simple cocoa/chocolate cake recipe. In Poland, we call this cake "a pickaninny" (in polish - murzynek). If you want to bake a bigger cake, scale up the ingredients properly, you might also want to increase baking time. Original recipe (provided to me by my girlfriend) had 1 cup of sugar in it and used oil instead of butter. But since i
+prefer butter, and after testing - less sugar was still perfectly fine in terms of sweetness (also, do you want the beetus?). Hence this recipe.
+
+- ⏲ Prep time: 15 min
+- 🍳 Cook time: 40-50 min
+- 🍽 Servings: depends
+
+## Ingredients
+- 2 cups flour
+- 0.5 cup of sugar
+- 1 tsp. baking soda (sodium bicarbonate)
+- 2 sp. cocoa powder
+- 2 whole eggs
+- 100 grams of butter
+- 1 cup fat milk
+- 3 sp. high quality jam of choice
+
+## Directions
+- Heat the oven to 180 C/356 F
+- Mix wet ingredients
+- Add dry ingredients
+- Mix until the consistence is nice and smooth
```

Comment:
```suggestion
- Mix until the consistency is nice and smooth
```

Reply:
How do I add pic to this pull request? Kinda new to github.